modules/candidate_parameters/jsx/ProbandInfo.js

```diff
@@ -97,6 +97,66 @@ var ProbandInfo = React.createClass(
         dob2Required = true;
       }
 
+      var extraParameterFields = [];
+      var extraParameters = this.state.Data.extra_parameters;
+      for (var key2 in extraParameters) {
+        if (extraParameters.hasOwnProperty(key2)) {
```

Comment:
Side note: this render function is getting pretty long. I'd break this logic out into another function:

```
var extraParametersFields = extraParameters.map(this._convertExtraParamToInputComponent);
```